scope.h

```diff
@@ -195,6 +195,12 @@ scope has the given name. C<name> must be a literal string.
 #define SAVEDESTRUCTOR_X(f,p) \
           save_destructor_x((DESTRUCTORFUNC_t)(f), (void*)(p))
 
+#define SAVEMORTALSVFUNC_X(f,sv) \
+          mortal_svfunc_x((SVFUNC_t)(f), sv)
+
+#define SAVEMORTALDESTRUCTOR_SV(coderef,args) \
+          mortal_destructor_sv(coderef,args)
+
```

Comment:
A small point on the naming: I don't think these should be called "SAVE...", because they don't push anything to the save stack. Every other "SAVEfoo" macro is a wrapper for "push some values to the save stack".

I like the idea of them, I just think the name is misleading.

Reply:
The match SAVEMORTALIZESV(), which where the naming comes from.

Reply:
Oh I see. Boo. In that case, all the names are annoying and we should fix them all up at some point in the future, but for now yours isn't making it (much) worse, so might as well stick to that precedent then.

Reply:
Well, @iabyn pointed out the flaw in my reasoning, so I have removed the SAVE from the macro wrappers as you requested originally. Sorry for not fully understanding SAVEMORTALIZESV() and confusing matters.